### DISCUSSION TITLE
CMD + F1 falsely reported as system shortcut

### DISCUSSION THREAD

**Discussion by L-i-v-y** [MonitorControl/MonitorControl#1150] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

CMD + F1 combination falsely reported as system-wide shortcut when it is not set as such in system preferences and blocked from being used with MonitorControl.

### Steps to reproduce

- Try to set CMD+F1 as shortcut for anything
- Error message "System-wide Keyboard Shortcut" appears
- Check system keyboard shortcuts for CMD + F1
- CMD + F1 is unused (neither active, nor inactive)
- Try to set CMD + F1 again
- Error message "System-wide Keyboard Shortcut" appears

### Expected behavior

- Try to set CMD + F1 as shortcut for anything
- CMD + F1 will be set as shortcut
##IF CMD + F1 is keyboard shortcut:
- Show error message "System-wide Keyboard Shortcut"

### Anything else?

CMD + F1 seems to have worked fine when testing Lunar for monitor controls — has been uninstalled in the meantime and the system was rebooted after removal.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.5
- Mac model: MacBook Pro (14-inch, 2021)
- MonitorControl version: 4.1.0
- Monitor(s): irrelevant
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Afaik the Command+F1 is the enable/disable mirroring shortcut and is indeed reserved.

But the issue is outside of MonitorControl's jurisdiction as the custom shortcuts implementation is provided by [KeyboardShortcuts](https://github.com/sindresorhus/KeyboardShortcuts). The message also comes from that package. I think you should raise the issue there.
